packages/edge-gateway/src/gateway.js

```diff
@@ -120,6 +120,19 @@ export async function gatewayGet(request, env, ctx) {
       })()
     )
 
+    // Block content types
+    if (
+      FORBIDDEN_CONTENT_TYPES.includes(
+        winnerGwResponse.response.headers.get('content-type')
+      )
+    ) {
+      throw new ForbiddenContentError(
+        `Forbidden content type: ${winnerGwResponse.response.headers.get(
+          'content-type'
+        )}`
+      )
```

Comment:
We need to do this check before the `waitUntil` above don't we? Otherwise we cache the response and on subsequent requests it'll be served from the cache.

Reply:
yeah good catch